.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+# Description
+
+This PR includes the following proposed change(s):
+
+- {List all the changes, if possible add the jira ticket #}
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Refactoring / Documentation
+
+if your change is a breaking change, please add `breaking change` label to this PR
+
+## How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes.
+
+## Does the change impact or break the Docker build?
+
+- [ ] Yes
+- [ ] No
+
+If Yes: Has Docker been updated accordingly?
+
+- [ ] Yes
+- [ ] No
```

Comment:
I honestly dont know the current status of the docker file in this project...XD

Reply:
good point, was copy pasta

Reply:
resolved